Quicksilver/Quicksilver.pkg.recipe.yaml

```diff
@@ -0,0 +1,14 @@
+Description: Downloads the current release version of Quicksilver and builds a package.
```

Comment:
You know I'm a fan of yaml recipes, but let's hold off on adding any to this repo until AutoPkg officially supports it.

Reply:
Removed